smooks-core/src/main/java/org/milyn/payload/JavaResult.java

```diff
@@ -42,7 +42,7 @@
      * Public default constructor.
      */
     public JavaResult() {
-        this(false);
+        this(true);
```

Comment:
on the contrary, it's more or less to give them some backward compatibility (previously hashmap behaves simillar to linkedhashmap )- this is not necessary and if you want I can bring back previous behaviour